src/data/actions/accessibility.test.jsx

```diff
@@ -16,19 +16,50 @@ const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
 let store;
 
-describe('Accessibility Action Creator', () => {
+describe('Accessibility Action Creators', () => {
+  beforeEach(() => {
+    store = mockStore(initialState);
```

Comment:
I'm pulling this out into a beforeEach, since each test case shares the same initial mock store state.